htdocs/xoops_lib/composer.dist.json

```
@@ -30,9 +30,11 @@
     "smarty/smarty": "^4.5",
     "smottt/wideimage": "^1.1",
     "tecnickcom/tcpdf": "^6.10",
```

Biome (2.5.0)

[error] The key tecnickcom/tcpdf was already declared.

(lint/suspicious/noDuplicateObjectKeys)